Clean up SDK (#23)
fragment-dev/fragment-go: README.md

	fmt.Println("Latest balance of account is: ", response.LedgerAccount.OwnBalance)
}
```

### Read an Account and Ledger's Schema

> To get a Ledger and Account Schema from your Workspace:

Reviewer: ```suggestion
### Read a Schema

To get a Schema from your Workspace:
```

``` go
package main

import (
	"encoding/json"
	"fmt"
	"os"

	"github.com/fragment-dev/fragment-go/queries"
)

func main() {
	data, err := queries.GetSchema(authenticatedContext, "test-schema", nil)
	if err != nil {
		fmt.Println("Failed to get schema.")
		fmt.Println(err)
		os.Exit(1)
	}

	// Marshal the entire response to pretty JSON
	jsonData, err := json.MarshalIndent(data.Schema.Version.Json, "", "  ")
	if err != nil {
		fmt.Printf("Error marshaling to JSON: %v\n", err)
		os.Exit(1)
	}

> 

Reviewer: ```suggestion
    # Save the Schema as a file in your repository
```

	err = os.WriteFile("fragment-schema.json", jsonData, 0644)
	if err != nil {
		fmt.Printf("Error writing file: %v\n", err)
		os.Exit(1)
	}
}
```

### Update an Account and Ledger's Schema

We recommend you update your schema directly in the Fragment Dashboard, but if you need to do this dynamically you can combine the example above and below to:

1. Get a JSON of your schema
2. Make updates as you need
3. Post your updated JSON

> To [post](https://fragment.dev/api-reference/api-mutations#storeschema) the updates to your Ledger and Account Schema:

Reviewer: ```suggestion
### Store a Schema

To [store](https://fragment.dev/api-reference/api-mutations#storeschema) a new version of your Schema:
```
